src/index.js

```diff
@@ -1,17 +1,17 @@
 const dict = new WeakMap();
+const isFrozenArray = val => Array.isArray(val) && Object.isFrozen(val);
+const isTemplateObject = val => isFrozenArray(val) && isFrozenArray(val.raw);
 
-export const doc = (...args) => {
-  const [strings, ...keys] = args;
-  if (Array.isArray(strings) && typeof strings.raw !== 'undefined') {
-
+export const doc = (first, ...keys) => {
```

Comment:
Just wondering: is there a reason for having both named and default export?

Reply:
I think yes. You able to have both names for one func in one module
```js
import describe, {doc} from '../src';

const user = describe
  `This is User object with id ${'id'}`
  (api.get('user', 1));

console.log(doc(user));
```